src/api/Action/Action.ts

```diff
@@ -25,7 +30,7 @@ export class Action {
   private constructor(
     private readonly _url: string,
     private readonly _data: ActionGetResponse,
-    private readonly _metadata: ActionMetadata,
+    private readonly _metadata: Partial<ActionMetadata>,
```

Comment:
shouldn't we change `ActionMetadata` type itself, to avoid using `Partial` later?